src/cmd/create.go

```diff
@@ -699,9 +700,14 @@ func pullImage(image, release, authFile string) (bool, error) {
 	}
 
 	if promptForDownload {
+		infoSize, err := skopeo.Inspect(imageFull)
+		if err != nil {
+			logrus.Debug("cannot get the size of the image")
+		}
+
 		fmt.Println("Image required to create toolbox container.")
 
-		prompt := fmt.Sprintf("Download %s (500MB)? [y/N]:", imageFull)
+		prompt := fmt.Sprintf("Download %s (%vMB)? [y/N]:", imageFull, infoSize)
```

Comment:
If we fail to get the image size, then this message will look odd:
```
Download <fully-qualified-image-name> [0MB]? [y/N]:
```

In many cases, it will fail because the `skopeo(1)` binary was absent from the system, which I think should be a [valid](https://github.com/containers/toolbox/issues/752) use-case.  So we should try to cover those situations elegantly.

Reply:
I tried to solve this by adding an else on the code. Check if it works for you or if it could be done in a more elegantly way.